app/models/user.rb

```diff
@@ -9,6 +9,14 @@ class User < ActiveRecord::Base
 =end
   has_many :posts
   has_many :comments
+  has_many :active_relationships,  class_name:  "Relationship",
+                                   foreign_key: "follower_id",
+                                   dependent:   :destroy
+  has_many :passive_relationships, class_name:  "Relationship",
```

Comment:
Active and passive seem like the wrong words. How about following and followed_by?